dev/com.ibm.ws.jaxrs.2.0_fat/build.gradle

```diff
@@ -46,17 +46,17 @@ dependencies {
     'org.codehaus.jackson:jackson-jaxrs:1.6.2',
     'org.codehaus.jackson:jackson-mapper-asl:1.6.2',
     'org.codehaus.jackson:jackson-xc:1.6.2'
-  jackson2x 'com.fasterxml.jackson.core:jackson-annotations:2.4.1',
-    'com.fasterxml.jackson.core:jackson-core:2.4.1',
-    'com.fasterxml.jackson.core:jackson-databind:2.4.1',
-    'com.fasterxml.jackson.jaxrs:jackson-jaxrs-base:2.4.1',
-    'com.fasterxml.jackson.jaxrs:jackson-jaxrs-json-provider:2.4.1',
-    'com.fasterxml.jackson.module:jackson-module-jaxb-annotations:2.4.1'
+  jackson2x 'com.fasterxml.jackson.core:jackson-annotations:2.11.2',
```

Comment:
would it be possible to replace all the individual jackson lib references with the one or two wrapper projects?  This way - any update in the future is in one place...   This goes for all gradle file refs.

would look like:  project(':io.openliberty.com.fasterxml.jackson'),

Reply:
done